src/charts/trends/trends-chart.directive.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.directive:pfTrendsChart
+ *
+ * @description
+ *   Directive for rendering a trend chart. The trend chart combines overall data with a pfCard and a
+ *   pfSparklineChart.
+ *   <br><br>
+ *   See http://c3js.org/reference.html for a full list of C3 chart options.<br>
+ *   See also: {@link patternfly.charts.directive:pfSparklineChart}
+ *
+ * @param {object} config configuration settings for the trends chart:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.chartId      - the unique id of this trends chart
+ * <li>.title        - title of the Trends chart
+ * <li>.timeFrame    - (optional) the time frame for the data in the pfSparklineChart, ex: 'Last 30 Days'
+ * <li>.units        - unit label for values, ex: 'MHz','GB', etc..
+ * </ul>
+ *
+ * @param {object} chartData the data to be shown in the sparkline charts<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.total  - number representing the total amount
+ * <li>.xData  - Array, X values for the data points, first element must be the name of the data
+ * <li>.yData  - Array, Y Values for the data points, first element must be the name of the data
+ * </ul>
+ *
+ * @param {int=} chartHeight   height of the sparkline chart
+ * @param {boolean=} showXAxis override sparkline config settings for showing the X Axis
+ * @param {boolean=} showYAxis override sparkline config settings for showing the Y Axis
+ * @example
+ <example module="patternfly.charts">
+ <file name="index.html">
+  <div ng-controller="ChartCtrl" class="row" style="display:inline-block; width: 100%;">
+   <div class="col-md-12">
+     <div pf-trends-chart config="config" chart-data="data" chart-height="custChartHeight"
+          show-x-axis="custShowXAxis" show-y-axis="custShowYAxis"></div>
+   </div>
+   <div class="col-md-12">
+   <div class="row">
+     <div class="col-md-4">
+       <form role="form"">
+         <div class="form-group">
+           <label>Show</label></br>
+           <label class="checkbox-inline">
+              <input type="checkbox" ng-model="custShowXAxis">X Axis</input>
+           </label>
+           <label class="checkbox-inline">
+             <input type="checkbox" ng-model="custShowYAxis">Y Axis</input>
+           </label>
+         </div>
+       </form>
+     </div>
+     <div class="col-md-4">
+       <form role="form" >
+         <div class="form-group">
+           <label>Chart Height</label></br>
+           <input style="height:25px; width:60px;" type="number" ng-model="custChartHeight"></input>
+         </div>
+       </form>
+     </div>
+     <div class="col-md-4">
+       <button ng-click="addDataPoint()">Add Data Point</button>
+     </div>
+   </div>
+   </div>
+  </div>
+ </file>
+
+ <file name="script.js">
+ angular.module( 'patternfly.charts' ).controller( 'ChartCtrl', function( $scope ) {
+
+       $scope.config = {
+         'chartId': 'exampleTrendsChart',
```

Comment:
I personally don't care how this is spaced but it should probably be consistent :)